### PR TITLE
- Release new version 0.6.0

### PR DESCRIPTION
**Pull request by rjschwei**
+ Move spec file to build as python3 code
  + Remove all references and conditions for SLE 11 build
  + Fix issue with Apache plugin only writing ACL to last configured file #1

### CODE REVIEW

File: python3-serviceAccessConfig.spec [hunk 1]
```diff
@@ -28,39 +28,31 @@ Source0:        serviceAccessConfig-%{version}.tar.bz2
...
-BuildRequires:  python-requests
-BuildRequires:  python-setuptools
+Requires:       python3-base
+Requires:       python3-docopt
+Requires:       python3-requests
+BuildRequires:  python3-mock
+BuildRequires:  python3-pytest
+BuildRequires:  python3-requests
+BuildRequires:  python3-setuptools
+BuildRequires: systemd
```
Comment (smarlowucf) [inline python3-serviceAccessConfig.spec]: Match tab for consistency.

File: python3-serviceAccessConfig.spec [hunk 2]
```diff
@@ -83,55 +75,32 @@ gzip %{buildroot}/%{_mandir}/man1/serviceAccessConfig.1
...
-%if 0%{?suse_version} > 1140
     %service_del_postun serviceAccessConfig.service
-%else
-    %insserv_cleanup
-%endif
-
 
 %files
 %defattr(-,root,root,-)
 %doc LICENSE README.md
```
Comment (smarlowucf) [inline python3-serviceAccessConfig.spec]: Move license to license macro? `%license LICENSE`

File: lib/serviceAccessConfig/apache.py [hunk 3]
```diff
@@ -169,10 +169,10 @@ def _update_service_config(self, cidr_blocks):
                 else:
                     new_content += ln
 
-        # Write the updated config
-        fout = open(cfg, 'w')
-        fout.write(new_content)
-        fout.close()
-        logging.info('Updated Apache config file %s' % cfg)
+            # Write the updated config
+            fout = open(cfg, 'w')
```
Comment (smarlowucf) [inline lib/serviceAccessConfig/apache.py]: Context managers FTW :wink: :

```
with open(cfg, 'w') as fout:
    fout.write(new_content)
```

**Review by smarlowucf — CHANGES_REQUESTED** [review]

**Review by smarlowucf — APPROVED** [review]

**Comment by rjschwei** [comment]
Requested changes implemented @smarlowucf is on vacation, merging.

**Pull request merged**